### ISSUE TITLE
feature request: show the party's skills as well

### ISSUE THREAD

**Issue by quisquous** [Rawrington/SkillDisplay#2]
This project looks great, and I'm excited to use this.  Do you have any plans to show the rest of the party's skills as well?  I think that would be really great for streaming.

**Comment by Rawrington** [comment]
Thanks! I could perhaps implement something like that, I'm currently refactoring the main App component to be a functional component and running into some stumbling blocks, but I'll put it on my todo!

**Comment by Rawrington** [comment]
My current issue with something like that is since I'm getting all the log lines I'd have to rely on ACT's party filter or manually filter party members, or make it configurable which users are shown

**Comment by jcheikha** [comment]
Hi, this looks nice and i can't wait to start using it, however i am facing issues with starting it. when i paste the url and run it, it shows this: http://0.0.0.0:10501/github/rawrington/SkillDisplay/?HOST_PORT=ws://0.0.0.0:10501/ in the manager window url.

**Comment by juliancalzada** [comment]
is this still in active dev? i followed the instructions but nothing is being displayed. the wiki is pretty sparse.

**Comment by Rawrington** [comment]
yes, I am just not personally checking this repo a lot and am slowly working on an update to add a bunch more features
